pom.xml

```diff
@@ -20,12 +77,54 @@
             <artifactId>javafx-fxml</artifactId>
             <version>13</version>
         </dependency>
-        <dependency>
-        	<groupId>org.jboss.as</groupId>
-        	<artifactId>jboss-as-controller-client</artifactId>
-        	<version>7.2.0.Final</version>
-        </dependency>
+ 
+ 		<!-- https://mvnrepository.com/artifact/org.wildfly.core/wildfly-controller-client -->
+		<dependency>
+		    <groupId>org.wildfly.core</groupId>
+		    <artifactId>wildfly-controller-client</artifactId>
+		    <version>15.0.26.Final-redhat-00001</version>
+		</dependency>
+		
+		
+		<!-- https://mvnrepository.com/artifact/org.wildfly.core/wildfly-core-management-client -->
+		<dependency>
+		    <groupId>org.wildfly.core</groupId>
+		    <artifactId>wildfly-core-management-client</artifactId>
+		    <version>15.0.26.Final-redhat-00001</version>
+		</dependency>
+		
+		<!-- https://mvnrepository.com/artifact/org.wildfly.core/wildfly-protocol -->
+		<dependency>
+		    <groupId>org.wildfly.core</groupId>
+		    <artifactId>wildfly-protocol</artifactId>
+		    <version>15.0.26.Final-redhat-00001</version>
```

Comment:
@theashiot just a total minor, you can parametrize the versions so you don't have to copy them and they can be found easily on the top, see for example wildfly.core pom.xml versions [here](https://github.com/wildfly/wildfly-core/blob/main/pom.xml#L184)

Reply:
done